app/views/spree/admin/shared/_head.html.haml

```diff
@@ -21,4 +21,7 @@
   = "jQuery.alerts.dialogClass = 'spree';"
   = raw "var AUTH_TOKEN = \"#{form_authenticity_token}\";"
 
+= render "layouts/bugherd_script"
```

Comment:
are we really using it? cc @RachL @kirstenalarsen

Reply:
yeah, same question for goggle analytics...

Reply:
Google Analytics is used by UK and Australia is considering switching to Matomo: https://openfoodnetwork.slack.com/archives/CG7NJ966B/p1562290349053500

However I'm sorry but I'm not sure what you are referring to @sauloperez

Reply:
that Bugherd script :point_right: http://www.bugherd.com

Reply:
We used Bugherd in the past. I will check with the team here.

Reply:
hey guys, this PR is just keeping Bugherd as it was, I'd say this is out of scope for this PR.
Because this PR is blocking the release, can we move it forward and handle Bugherd in a separate PR?